ansible/roles/repo/tasks/redhat.yaml

```diff
@@ -1,15 +1,18 @@
 ---
+- set_fact:
+   rhsm_user: "{{ lookup('env', 'RHSM_USER' ) | ternary (lookup('env', 'RHSM_USER' ), rhsm_user) }}"
```

Comment:
the `ternary` function's input is boolean. The `lookup` function returns String. 
ansible might be handling the condition internally. if not, we might need `lookup('env', 'RHSM_USER' ) != ""`